runsc/cmd/gofer.go

```diff
@@ -286,11 +286,15 @@ func setupRootFS(spec *specs.Spec, conf *config.Config) error {
 		// Prepare tree structure for pivot_root(2).
 		os.Mkdir("/proc/proc", 0755)
 		os.Mkdir("/proc/root", 0755)
+		os.Mkdir("/proc/etc", 0755)
```

Comment:
Can we check the returned error here (and for the two directly above `os.Mkdir` calls)?

Reply:
Sure.